.github/workflows/gradle-release.yml

```
@@ -0,0 +1,62 @@
+
+      - name: Git Config
+        run: |
+          git config user.name 'Nya Ξlimu'
+          git config user.email 'info@elimu.ai'
+
+      - name: Remove -SNAPSHOT from versionName
+        run: ./gradlew removeSnapshot
+
+      - name: Set the new versionName as an environment variable
+        id: get_version_name
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)